### DISCUSSION TITLE
I Need Help

### DISCUSSION THREAD

**Discussion by ManuelKlimek** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#725] [General]
Roidmi eve plus
Rooms/Pin&go/Zone cleanup is not working
and I don't understand why
The service could not be called Entity vacuum.roidmi v60 f86c_robot cleaner does not support this service.

![Screenshot 2024-04-07 102956](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/91559372/9fa2db27-a10b-466f-94e9-30f1a484c0da)

**Comment by sca075** [comment]
@ManuelKlimek  can you share the card configuration and the type of integration you use for your vacuum?

**Reply by ManuelKlimek** [reply]
type: custom:xiaomi-vacuum-map-card
map_source:
  camera: camera.xiaomi_cloud_map_extractor
calibration_source:
  camera: true
entity: vacuum.roidmi_v60_f86c_robot_cleaner
vacuum_platform: send_command
map_modes:
  - template: vacuum_clean_zone
  - template: vacuum_goto
  - template: vacuum_clean_segment
    predefined_selections:
      - id: '1'
        icon:
          name: mdi:broom
          x: 1249.9996442347765
          'y': -5000.00043399632
        label:
          text: Wohnzimmer
          x: 1249.9996442347765
          'y': -5000.00043399632
          offset_y: 35
        outline:
          - - -1025.0003896653652
            - -6975.000463426113
          - - 3524.999678134918
            - -6975.000463426113
          - - 3524.999678134918
            - -3025.000404566526
          - - -1025.0003896653652
            - -3025.000404566526
      - id: '2'
        icon:
          name: mdi:broom
          x: 1299.9996449798346
          'y': -1075.0003755092619
        label:
          text: Essplatz
          x: 1299.9996449798346
          'y': -1075.0003755092619
          offset_y: 35
        outline:
          - - -875.0003874301909
            - -2975.000403821468
          - - 3474.99967738986
            - -2975.000403821468
          - - 3474.99967738986
            - 824.9996528029445
          - - -875.0003874301909
            - 824.9996528029445
      - id: '3'
        icon:
          name: mdi:broom
          x: -2500.000411644578
          'y': -5150.000436231494
        label:
          text: Flur
          x: -2500.000411644578
          'y': -5150.000436231494
          offset_y: 35
        outline:
          - - -3975.0004336237907
            - -7325.00046864152
          - - -1025.0003896653652
            - -7325.00046864152
          - - -1025.0003896653652
            - -2975.000403821468
          - - -3975.0004336237907
            - -2975.000403821468

**Reply by ManuelKlimek** [reply]
![grafik](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/91559372/080df6a1-48da-4c75-9f5d-c2e1465eadca)

**Reply by ManuelKlimek** [reply]
![grafik](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/91559372/7cbab04c-a44a-40eb-ac79-4229e1dbc8ae)

**Reply by ManuelKlimek** [reply]
![grafik](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/91559372/d41a90bd-d977-4a55-8561-70b474d80875)

**Reply by ManuelKlimek** [reply]
![grafik](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/91559372/80fa2d8f-b8d0-41c6-8929-9bd6d8a4ec23)

**Reply by ManuelKlimek** [reply]
![grafik](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/91559372/a025af75-8095-470a-b846-52d40b27f1ac)

**Comment by sca075** [comment]
@ManuelKlimek  the platform selected in the card is incorrect.

vacuum_platform: send_command probably will not work, https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/blob/master/docs/templates/alOneHassXiaomiMiot.md

this should be the one to use in your case.

**Reply by ManuelKlimek** [reply]
actually default
but the same error occurs everywhere

**Comment by PiotrMachowski** [comment]
You use Xiaomi Miot integration that does not work out of the box and needs config specific to vacuum model.

https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/527
https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/618
